bindings/ros/README.md

```diff
@@ -33,64 +33,4 @@ sudo cmake --build . --target install
 
 # 3. Usage
 
-In directory ```catkin_ws/src/``` clone the repository:
-
-```console
-git clone https://github.com/analogdevicesinc/tof-ros
-```
-
-After cloning the repository in the ``catkin_ws/ run the following command:
- 
-```console
-catkin_make
-source devel/setup.bash
-```
-
-### Starting the camera node 
-
-|                                 |          |   |
-|---------------------------------|----------|---|
-| With RQT Dynamic reconfigure    | USB      |`roslaunch aditof_roscpp camera_node_gui.launch config_file:="<path>"`   |
-|                                 | Ethernet |`roslaunch aditof_roscpp camera_node_gui.launch ip:="10.42.0.1" config_file:="<path>"`   |
-| Without RQT Dynamic reconfigure | USB      |`roslaunch aditof_roscpp camera_node.launch config_file:="<path>" mode:="1" use_depthCompute:="false"`   |
-|                                 | Ethernet |`roslaunch aditof_roscpp camera_node.launch ip:="10.42.0.1" config_file:="<path>  mode:="1" use_depthCompute:="false"`   |
-
-
-### Parameters:
- [config_file:"<<b>path></b>"]
-* Crosby with Pulsatrix: "```config/config_walden_3500_nxp.json```"
-* Crosby: "```config/config_crosby_nxp.json```"
-* Walden: "```config/config_walden_nxp.json```"
-
- [use_depthCompute] 
- - `true` for enabling Depth Compute libraries
- - `false` for disabling Depth Compute libraries 
-
- [mode]:
-* `1` -> QMP mode of the camera
-* `2` -> MP mode of the camera
-
-###  Dynamic reconfigure window:
-
-    
- <p align="center"><img src="../../doc/img/ros_dynamic_reconfigure.png" /></p>
- 
-
-### Examples
-  - Visualize point cloud in rviz
-    ```console
-    cd catkin_ws
-    source devel/setup.bash
-    roslaunch aditof_roscpp rviz_publisher.launch
-    ```
-
-
-### Published Topics
-The aditof_camera_node publishes messages defined by the [sensor_msgs](http://wiki.ros.org/sensor_msgs) package on the following topics
-- /aditof_roscpp/aditof_camera_info
-- /aditof_roscpp/aditof_depth
-- /aditof_roscpp/aditof_ir
-- /aditof_roscpp/aditof_pcloud
-
-### Update parameters at runtime using
-Using the [dynamic_reconfigure](http://wiki.ros.org/dynamic_reconfigure) package, the aditof_ros_package offers the users the possibility to update the camera parameters
+For usage information please refere to ```section 3``` from: [ROS Usage](https://github.com/analogdevicesinc/tof-ros)
```

Comment:
Can you use the direct link to section 3 instead? It's easier for the users to find it.
https://github.com/analogdevicesinc/tof-ros2#3-usage